modules/tinymce/src/themes/silver/test/ts/browser/skin/OxideTablePickerMenuTest.ts

```diff
@@ -13,7 +13,8 @@ const tableCellsApprox = (s: ApproxStructure.StructApi, str: ApproxStructure.Str
     for (let j = 1; j <= 10; j++) {
       cells.push(s.element('div', {
         attrs: {
-          role: str.is('button')
+          role: str.is('button'),
+          ['aria-label']: str.is(`${j} columns, ${i} rows`)
```

Comment:
Would it be better to announce the row first?

Reply:
The spec calls for columns then rows, not that it bothers me.